docs/community/SIGs/Runtime/SIG-Runtime-CHARTER.md

```diff
@@ -0,0 +1,139 @@
+# SIG Runtime — Charter
+
+**Status:** Proposed  
+**Last updated:** 2025-10-01
+
+## Mission & Scope
+
+### Mission
+
+The SIG Runtime maintains and shapes:
+
+- **OCM Language Bindings** ([bindings](https://github.com/open-component-model/open-component-model/tree/main/bindings))
+- **OCM CLI** ([cli](https://github.com/open-component-model/open-component-model/tree/main/cli))
+- **OCM Kubernetes Controller** ([kubernetes/controller](https://github.com/open-component-model/open-component-model/tree/main/kubernetes/controller))
+- **Legacy version of OCM Library and CLI (only maintenance)** ([ocm](https://github.com/open-component-model/ocm/tree/main))
+
+The mission of SIG Runtime is to allow OCM components to be:
+
+- creatable
+- signable
+- transportable
+- verifiable
+- deployable
+
+### Scope
+
+- **OCM Language Bindings**
+  - Modular, extensible runtime library for components and versions
+  - Support for dynamic creation of component versions
+  - Standards-compliant signing and verification per the [OCM Specification](https://github.com/open-component-model/ocm-spec/tree/main)
+  - Introspection, transport, and transformation of components and versions
+
+- **OCM CLI**
+  - Unified toolchain to create, inspect, sign, verify, and transport components and versions
```

Comment:
"component and versions" ? is this correct? wouldn't it be component versions...?